### PR TITLE
[Feature Request] Add 2 homescreen.

### PR DESCRIPTION
**Pull request by heygema**
# Description
Smaller to #3 which added only manifest file for add to homescreen feature. though I'm not sure about the provided icon.

- [x] added manifest.json and icon files (sample)

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "name": "KabarVirus.com",
+    "short_name": "Kabar Virus",
+    "start_url": "/?utm_source=homescreen",
```
Comment (ariya) [inline manifest.json]: Since we don't use GA, the "utm" part is not needed.

File: manifest.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    "lang": "id",
+    "display": "standalone",
```
Comment (ariya) [inline manifest.json]: I think it has to be `minimal-ui` since there are some possible back/forth navigations.

File: manifest.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    "background_color": "#FFFFFF",
+    "theme_color": "#536878",
+    "description": "KabarVirus.com menyajikan perkembangan COVID-19 di Indonesia dengan penampilan ringan dan sederhana agar tidak menghamburkan kuota data. Kode sumber: github.com/ariya/kabarvirus.",
```
Comment (ariya) [inline manifest.json]: Feel free to shorten it further.

File: src/generate.js [hunk 4]
```diff
@@ -129,6 +143,11 @@ function generate() {
     mkdirp('public');
     fs.writeFileSync('public/index.html', indexHtml);
 
+    copyFiles([
```
Comment (ariya) [inline src/generate.js]: Probably a bit too complicated: check the use of `copyFileSync` elsewhere in this file.

**Comment by ariya** [comment]
Also, better to place the new files in `template/`, just like the existing ones.

**Comment by heygema** [comment]
anyway this is the icon preview, so no one gets mad:
kvicon

previously made this one (gimp is hard):
kabarvirus

**Review by ariya — APPROVED** [review]

**Comment by ariya** [comment]
Awesome. Thank you 👍

**Pull request merged**